site/client/SearchCountry.tsx

```diff
@@ -24,21 +24,26 @@ const CovidSearchCountry = () => {
                 const country = asArray(selected)[0].value
                 Analytics.logCovidCountryProfileSearch(country)
                 setIsLoading(true)
-                window.location.href = `${covidCountryProfileRootPath}/${country}`
+                window.location.href = `${props.countryProfileRootPath}/${country}`
             }}
             isLoading={isLoading}
             placeholder="Search for a country..."
         />
     )
 }
 
-export function runCovidSearchCountry() {
-    const elements = Array.from(
-        document.querySelectorAll(".wp-block-covid-search-country")
-    )
-    elements.forEach(element => {
-        ReactDOM.render(<CovidSearchCountry />, element)
+export function runSearchCountry() {
```

Comment:
I know we don't do this yet, but I was thinking for all of these `run` methods, maybe we could have unit tests (or storybooks) to ensure we have some execution of these code paths and also to have some live documentation. Also would be nice to have some infra setup when things go wrong for regression testing (like what happened with multiple data explorers on the same page).

Reply:
Definitely something we need. Input would be an HTML page with selectors.